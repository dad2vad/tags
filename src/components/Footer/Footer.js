import React, { PureComponent } from 'react';
import Link from '../Link';
import config from '../../config.json';

class Footer extends PureComponent {
  render() {
    return (
      <footer className="footer">
        <div className="container">
          <div className="content has-text-centered">
            <p>
              <strong>{config.title}</strong> by
              <Link url="http://landish.github.io/" text="Lado Lomidze" />
            </p>
            <p>
              Built with
              <Link url="https://reactjs.org" text="React" />
              &nbsp;&amp;
              <Link url="http://bulma.io/" text="Bulma" />
            </p>
            <p>&copy; 2016-2017</p>
          </div>
        </div>
      </footer>
    );
  }
}

export default Footer;
