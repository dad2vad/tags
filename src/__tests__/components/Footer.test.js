import React from 'react';
import { shallow } from 'enzyme';
import Footer from '../../components/Footer';

describe('Footer', () => {
  it('renders correctly', () => {
    const component = shallow(<Footer />);
    expect(component).toMatchSnapshot();
  });
});
