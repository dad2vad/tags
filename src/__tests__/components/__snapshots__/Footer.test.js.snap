// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer renders correctly 1`] = `
<footer
  className="footer"
>
  <div
    className="container"
  >
    <div
      className="content has-text-centered"
    >
      <p>
        <strong>
          Comparison of Front-End Frameworks
        </strong>
         by
        <Link
          text="Lado Lomidze"
          url="http://landish.github.io/"
        />
      </p>
      <p>
        Built with
        <Link
          text="React"
          url="https://reactjs.org"
        />
         &
        <Link
          text="Bulma"
          url="http://bulma.io/"
        />
      </p>
      <p>
        © 2016-2017
      </p>
    </div>
  </div>
</footer>
`;
